src/psyclone/transformations.py

```diff
@@ -1353,9 +1426,14 @@ def apply(self, loop, depth=None):
 
         if loop.loop_type == "":  # iteration space is cells
             loop.set_upper_bound("cell_halo", depth)
-        else:  # iteration space is dofs
+        elif loop.loop_type == "colour":
```

Comment:
Please add comment, e.g. `# Loop is over cells of a single colour`

Reply:
OK, done.